openedx/core/djangoapps/catalog/management/commands/tests/test_create_catalog_integrations.py

```diff
@@ -5,13 +5,22 @@
 from django.test import TestCase
 from django.core.management import call_command, CommandError
 
+from edx_django_utils.cache.utils import TieredCache
+
 from openedx.core.djangoapps.catalog.models import CatalogIntegration
 from openedx.core.djangoapps.catalog.tests.mixins import CatalogIntegrationMixin
 
 
 class TestCreateCatalogIntegrations(CatalogIntegrationMixin, TestCase):
     """ Test the create_catalog_integrations command """
 
+    def tearDown(self):
+        """
+        After each test, flush out the cache
+        """
+        super(TestCreateCatalogIntegrations, self).tearDown()
+        TieredCache.delete_all_tiers(CatalogIntegration.cache_key_name())
```

Comment:
Can it instead inherit `CacheIsolationMixin` so that it automatically clears all cache entries after running?

Reply:
Interesting. Learnt a new test tool! Feedback incorporated.